build-tools/packages/build-tools/src/common/npmPackage.ts

```diff
@@ -123,7 +123,7 @@ export class Package {
 	 * The name of the package including the scope.
 	 */
 	public get name(): string {
-		return this.packageJson.name;
+		return this.isReleaseGroupRoot ? `[group]${this.monoRepo!.kind}` : this.packageJson.name;
```

Comment:
```suggestion
		return this.isReleaseGroupRoot ? this.monoRepo!.kind as string : this.packageJson.name;
```

Is prepending [group] to it necessary? I'd rather just rename the root packages to be something like "client-release-group" and not special-case them here.

Reply:
Yeah, I have considered that, then we should just use the name in `package.json`  instead.  I will revert this change totally.